test/testdrive/ci-only/kinesis.td

```diff
@@ -59,3 +59,18 @@ test strings
 "and a third!"
 "two more"
 "test strings"
+
+# Test that the ChainProvider is grabbing credentials if not statically provided.
```

Comment:
This is important to test! Blocked by: https://github.com/MaterializeInc/materialize/issues/2895